app/models/search.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Search
```

Comment:
Should be fine as we are just searching on planning applications for now but wonder if we could make this more agnostic and less coupled with planning applications as we add different search criteria in . Or we specify that this search class is just for planning applications

Reply:
I think I like the second idea...making this `PlanningApplicationSearch`. That should push us to create separate models if we need to search other tables!

Reply:
I think this should be a service called `PlanningApplicationSearch`, which takes some arguments and search for a planning application and returns it

Reply:
Yes, that's what I'm thinking too 😄

Reply:
Updates coming up!